Makefile

```diff
@@ -25,7 +25,7 @@ fuseml: generate lint build
 # Generate code, run linter, build FuseML release-ready archived binaries for all supported ARCHs and OSs
 release: test release_all
 
-build: build_server build_client
+build: generate build_server build_client
```

Comment:
Why this? I thinnk the point of leaving out `generate` was to speed up local builds when you do not need to regenerate the code...

Reply:
Well, generate is required on a fresh environment like CI.

You can also just build with the `build_server` and `build_client` targets. If you think that this is too verbose, I could also call the generate on the ci job.

Reply:
Or you can use `fuseml` target from CI

Reply:
`fuseml` also runs `lint` which is also called by the `test` target.

So, if I use `fuseml` both jobs will perform lint, which is not necessary, and in case of failure both will fail and two of the same errors will be shown by github, which does not look good.

Reply:
OK. I agree on keeping the previous `build` behaviour so I removed the change to the Makefile and instead added generate to the job step.